apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/jvm/JVMService.java

```diff
@@ -104,8 +106,16 @@ public void run() {
             jvmBuilder.setClazz(ClassProvider.INSTANCE.getClassMetrics());
 
             sender.offer(jvmBuilder.build());
+
+            // refresh cpu usage percent
+            cpuUsagePercent = jvmBuilder.getCpu().getUsagePercent();
```

Comment:
I think `getUsagePercent` again would cause an issue. Have you tested?

Reply:
I have tested it using debug, it returned a val less than 100.

Reply:
What issue do you think will cause, maybe my debug way miss it.

Reply:
You are using a `#build`-ed builder to read data. I think it is much safe to read it from `build` result or create a local variable to host the value.